src/test/java/io/confluent/kafkarest/mock/MockTime.java

```diff
@@ -25,11 +25,18 @@
 
   @Override
   public long milliseconds() {
+    // Incrementing currentMs prevents tests from locking up
+    // "waiting" for the time checks in the code.  Without this
+    // the ConsumerManager never completed (it never calls sleep or waitOn)
+    currentMs++;
```

Comment:
Because the ConsumerReadTask only calls the milliseconds method failure to do something with the currentMs results in an infinite loop.  Incrementing here is somewhat arbitrary but at least the code doesn't get stuck forever...